server/proxy/install.go

```diff
@@ -68,6 +68,20 @@ func (p *Proxy) InstallApp(in Incoming, cc apps.Context, appID apps.AppID, deplo
 		defer icon.Close()
 	}
 
+	// See if the app is inaaccessible. Call its ping path with nothing
+	// expanded, ignore 404 errors coming back and consider everything else a
+	// "success".
+	//
+	// Note that this check is often ineffective, but "the best we can do"
+	// before we start the diffcult-to-revert install process.
+	_, err = p.callApp(in, *app, apps.CallRequest{
+		Call:    apps.DefaultPing,
```

Comment:
I'm wondering if the additional concept of a "ping path" is needed here. There are other known paths like `/bindings` that can be used. Any errors returned from there can be ignored. But maybe developers won't expect the path to be used in that way.

Reply:
I tried... the problem is, binding may have an expand and expect the data; the OAuth example app would crash if the `ActingUser` is not there... It does not seem proper to invoke /bindings without a full context, and the full context can not be properly expanded until the app is installed (sad!) so it's a chicken/egg situation.